Felienne/hedy
# [BUG FIX] Adds missing 'program is already running' message

**Description**
In this PR we add a missing error message, resulting in an empty modal alert. This only occurs in the situation where a user is currently running a program with some user input. While the inline modal is up the run button is disabled, when pressing it an error message should be returned "you already have a running program". This message was missing and is implemented in this PR.

**Fixes**
This PR fixes #2568

**How to test**
Create a program that requires some user input. When the inline modal is up, try running the program again and verify that an error message is correctly display with an explanation why this is not possible.
